python/lsst/ts/logging_and_reporting/utils.py

```diff
@@ -259,3 +292,19 @@ def spacify(name):
 
     column_map = {colname: spacify(colname) for colname in df.columns}
     return df.rename(columns=column_map)
+
+
+def get_access_token():
+    """Return access token to be sent in headers as Auth Bearer"""
+    try:
+        import lsst.rsp.utils
+
+        return lsst.rsp.utils.get_access_token()
+    except ImportError:
+        return os.getenv("ACCESS_TOKEN", None)
+
+
+def get_auth_header():
+    """return dict obj for request auth headers"""
+    toke = get_access_token()
+    return {"Authorization": f"Bearer {toke}"}
```

Comment:
So this utility function will only work when running notebooks from the RSP. The authentication will be different when having our final app running, so just commenting to keep that in mind when progressing in the development.